addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -5,28 +5,28 @@
     <!-- Add price information on tickets (multi tickets, aka in collapse) -->
     <xpath expr="//div[hasclass('o_wevent_registration_multi_select')]" position="inside">
         <t t-if="ticket.price">
-            <t t-if="(ticket.price - ticket.price_reduce) &gt; 1 and website.pricelist_id.discount_policy == 'without_discount'">
-                <del class="text-danger me-1" t-field="ticket.price" t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.pricelist_id.currency_id}"/>
+            <t t-if="(ticket.price - ticket.price_reduce) &gt; 1 and website.pricelist_id and website.pricelist_id.discount_policy == 'without_discount'">
```

Comment:
FYI `record.empty_relational.field = False`, which means it would still work without your change.
(keep the change, it's still clearer like that :ok_hand:)